src/main/java/com/rultor/profiles/Profiles.java

```diff
@@ -83,28 +81,23 @@ public Profile fetch(final Talk talk) throws IOException {
      * Fetch a profile from an XML.
      * @param xml The XML
      * @return Profile found
+     * @todo #1039:30min Read .rultor.yml for authentication purposes from
```

Comment:
@xupyprmv I don't think it is good idea to create puzzle for that. We will have broken security before solution for this puzzle is merged